src/views/SpaceDelegates.vue

```diff
@@ -270,6 +271,19 @@ onMounted(() => {
     </template>
     <Teleport to="#modal">
       <SpaceDelegatesDelegateModal
+        v-if="space.delegationPortal != null"
+        :open="route.query.delegate !== undefined"
+        :space="space"
+        :address="(route.query.delegate as string) || ''"
+        @close="handleCloseModalDelegate"
+        @reload="loadDelegates(matchFilter)"
+      />
+      <SpaceDelegatesSplitDelegationModal
+        v-if="
```

Comment:
This is opening both modal when the space has both set a delegationPortal and split-delegation strategies

Reply:
fixed in [18e49c7](https://github.com/snapshot-labs/snapshot/pull/4492/commits/18e49c74c87feea8d7e4030441dcea6ff532ad34)

I made it so if a split-delegation configuration of the strategy is present, it only shows the split-delegation modal (and only pulls from the split-delegation api for displaying voting power).

This made sense to me, as the split-delegation setup requires more specific setup, let me know if you would rather display an error when both a compound governor setup and a split-delegation strategy are configured.